src/test/ui/target-feature-wrong.rs

```diff
@@ -35,6 +35,31 @@ fn bar() {}
 mod another {}
 //~^ NOTE not a function
 
+#[target_feature(enable = "sse2")]
```

Comment:
Drive-by: can we move the test to not be right in `src/test/ui/` ; that's ungreat organization :)

Reply:
Fixed, thanks.